packages/core/src/actors.ts

```diff
@@ -253,7 +253,7 @@ export function fromObservable<
         observable = lazyObservable();
         subscription = observable.subscribe({
           next: (value) => {
-            sendNext(toSCXMLEvent(value, { origin: actorContext.self }));
+            sendNext(value);
```

Comment:
if `value` is not being boxed here in an event - why the same change has not been done to `sendError`?

Reply:
It should be; I will make that change (didn't get to it yet 😅)

Reply:
Fixed: [82fc286](https://github.com/statelyai/xstate/pull/3148/commits/82fc2860d836c3b90af7280913d1db8819d50a98)